src/content/content-events.js

```diff
@@ -7,12 +7,15 @@ import {
   READY_EVENT,
   REQUEST_PACKAGE_INFO_EVENT,
   RESPONSE_PACKAGE_INFO_EVENT,
+  EVENT_URL_CHANGED,
 } from '../events-shared';
 import * as storage from '../storage';
 
 const sendPackageInfoToWebapp = (info) => dispatchEvent(RESPONSE_PACKAGE_INFO_EVENT, info);
+const sendRealodMessageToWebapp = (domain) => dispatchEvent(EVENT_URL_CHANGED, domain);
```

Comment:
I'm not sure you need to send the URL_CHANGE message to the WebApp.

The only effect of the URL change is in the `content`, when the `content.npm.js` is reloading the `overlay-package-report`.

Reply:
You are right, I'll rename this method. The `URL_CHANGE` event is not sent to the web app but to the content-script.

Reply:
But the `content-events.js` is already the Content Script, why do you need to send another message to itself?

Reply:
So you suggest that I'll just call the method `loadPackageInfo` instead?

Reply:
https://github.com/os-scar/overlay/pull/116#discussion_r1233977008